setup.py

```diff
@@ -121,7 +121,11 @@ def read(*names, **kwargs):
         'open source', 'scan', 'license', 'package', 'dependency',
         'copyright', 'filetype', 'author', 'extract', 'licensing',
     ],
-    python_requires='>=2.7,<3',    
+    python_requires='>=2.7,<4',    
+    dependency_links = ['https://github.com/Abhishek-Dev09/libarchive/raw/master/extractcode-libarchive-3.1.2.tar.gz#egg=extractcode-libarchive-3.1.2',
```

Comment:
I cannot merge things that would work with a dependency link that is off your repository. If you modified somehow these plugins... why not make this a PR? the code is under `/plugins`

Reply:
I do not have push access to your repository.Thats why i have uploaded on my repo.
No need to worry i have built from /plugins only.